core/tools/migrate_db/README.md

```diff
@@ -1,26 +1,18 @@
-After updating `river_migrate_db.env` with correct parameters, migration tool can be run like this:
+### Build migration tool
 
-    # source DB still can be in use at this point
-    go build -o river_migrate_db .
-    ./river_migrate_db help
-    ./river_migrate_db source test  # test source config
-    ./river_migrate_db target test  # test target config
-    ./river_migrate_db target create  # create target schema (i.e. db partition)
-    ./river_migrate_db target init  # create targe tables
-    ./river_migrate_db target partition  # create target partitions matching source
+Update `river_migrate_db.env` with correct parameters.
 
-    # shutdown container that is connected to source db
+Source DB still can be in use at this point.
 
-    ./river_migrate_db target partition  # create missing target partitions matching source since last run
-    ./river_migrate_db copy  # copy data from source to target
+go build -o river_migrate_db .
 
-    # Inspect specific stream content
-    ./river_migrate_db source inspect <streamId>
-    ./river_migrate_db target inspect <streamId>
+./river_migrate_db help
+./river_migrate_db source test # test source config
+./river_migrate_db target test # test target config
 
-    # reconfigure container to use target db
+Note: shutdown node process connected to source DB during migration process below.
```

Comment:
You actually don't need to shut down the node for the `target create` and `target init` commands, but if it makes the docs easier to read, this isn't the biggest deal.